pages/common/cowsay.md

```diff
@@ -0,0 +1,15 @@
+# cowsay
+
+> Generate an ASCII character like a cow or sheep saying or thinking something
+
+- Print an ASCII cow saying "Hello world!"
+
+`cowsay {{"Hello world!"}}`
+
+- Print an ASCII dragon saying "Hello!"
+
+`echo "Hello!" | cowsay -f dragon`
+
+- Print a stoned thinking ASCII cow
+
+`cowthink -s {{"I'm just a cow, not a great thinker ..."}}`
```

Comment:
Maybe we should call them placeholders rather than tokens (it's more self-explanatory and intuitive for non-native English speakers)